memeify/memeify.py

```diff
@@ -0,0 +1,60 @@
+from redbot.core import Config, commands
+import re
+
+
+class Memeify(commands.Cog):
+    """Makes things memey."""
+
+    def __init__(self, bot):
+        super().__init__()
+
+        self.config = Config.get_conf(self, identifier=2934875294)

```

Comment:
should add `force_registration=True` to the get_conf call.

This makes sure you don't mistype a config variable name or anything, as you have to register variables in the config with the register_conf before using them.